languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -49144,8 +49147,9 @@ USA
                 <token regexp="yes">inferior|superior</token>
                 <token>than</token>
             </pattern>
-            <message>Some adjectives derived from Latin act as both adjective and comparative. In these cases use the collocation 'to'</message>
-              <suggestion>\1 to</suggestion>
+            <!-- -->
+            <message>Some adjectives derived from Latin act as both adjective and comparative. In these cases use the collocation 'to'.</message>
```

Comment:
Too technical. We should shorten it to "The adjective '\1' typically requires the preposition 'to'."